### PR TITLE
changed ingestion error behaviour

### PR DESCRIPTION
**Pull request by deepak1725**
Changed Ingestion Error Behaviour, No Error is thrown when Ingestion is disabled via Flag

Jira: https://issues.redhat.com/browse/APPAI-1454

Signed-off-by: Deepak Sharma <deepshar@redhat.com>

### CODE REVIEW

**Comment by prashbnair** [comment]
@deepak1725 From the bug, I understand that we are not returning any data even when there are packages that are present. Is that correct?

**Comment by deepak1725** [comment]
> @deepak1725 From the bug, I understand that we are not returning any data even when there are packages for which we have data. Is that correct?

@prashbnair Previously It was like:
- If there are unknown Packages and INGESTION is DISABLED, we raise an error. Like as mentioned in Bug.

So this PR changes this behavior to:
- If there are unknown Packages and then check if INGESTION is enabled only, trigger Unknown Flow.
No Error, like in Previous case is raised now.

File: bayesian/api/api_v2.py [hunk 1]
```diff
@@ -245,15 +242,17 @@ def post():
...
-            if os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW") == "1":
-                # Unknown Packages is Present and INGESTION is DISABLED
-                logger.error(ingestion_disabled_msg)
-                raise HTTPError(400, ingestion_disabled_msg)
-            unknown_package_flow(ecosystem, unknown_pkgs)
+
             if not stack_recommendation:
                 # If No Package is Known, all are unknown.
-                logger.debug(no_package_available_msg)
+                logger.debug(stack_recommendation)
```
Comment (prashbnair) [inline bayesian/api/api_v2.py:248]: will this not print "None"

Comment (deepak1725) [reply bayesian/api/api_v2.py]: Ohh Yeah.. missed it. Fixed.

**Review by prashbnair — APPROVED** [review]

**Pull request merged**